packages/common/random-access-storage/src/common/storage.ts

```diff
@@ -9,6 +9,9 @@ export enum StorageType {
   RAM = 'ram',
   IDB = 'idb',
   CHROME = 'chrome',
+  /**
+   * @deprecated
+   */
   FIREFOX = 'firefox',
   NODE = 'node',
   WEBFS = 'webfs',
```

Comment:
this is going to be deprecated too, right?

Reply:
yes, eventually